docs/billing.md

```diff
@@ -1,5 +1,51 @@
 # Billing
 
+Before diving into billing let's do a recap of some parts of important parts of the system.
+
+Note: During the rest of the documentation we'll consider ChainA as source chain and chainB as destinationChain.
+
+
+## On Chain
+
+We have multiple contracts interactions that happens to send message from ChainA to ChainB. These contracts are deployed on each chain that need to support CCIP
+1. Router: Non changing initial contract that users/dApps call to send a message
```

Comment:
these contracts should be well-defined enough in other places, should be unnecessary to redefine them in billing doc

Reply:
Router: Immutable entry point contract that users/dapps call to get quotes and send messages